terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
```

Comment:
This defaults to `true`, but never hurts to set for explicitness: https://www.terraform.io/docs/providers/aws/r/db_instance.html#auto_minor_version_upgrade.